integration/theia/extension/src/node/glsp-server-contribution.ts

```diff
@@ -17,9 +18,19 @@ export class IvyGLSPServerContribution extends GLSPSocketServerContribution {
       executable: '',
       additionalArgs: ['--consoleLog', 'false', '--fileLog', 'true', '--logDir', SERVER_DIR],
       socketConnectionOptions: {
-        port: getPort(PORT_ARG_KEY, DEFAULT_PORT)
+        port: getPort(PORT_ARG_KEY, DEFAULT_PORT),
+        host: getHost(HOST_ARG_KEY)
       },
       launchedExternally: true
     };
   }
 }
+
+function getHost(argsKey: string): string | undefined {
+  argsKey = '--' + argsKey + '=';
+  const args = process.argv.filter(a => a.startsWith(argsKey));
```

Comment:
I think you can also use process.env.IVY_PROCESS_GLSP_HOST to get the host

Reply:
@ivy-lli note that process.env does not work because host and port are passed as command line arguments. getHost(...) is more or less copy&paste from getPort(...).

Reply:
ah ok